x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/translations.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const EXCEPTION_LIST_ID_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.all.exceptions.idTitle',
+  {
+    defaultMessage: 'List ID',
+  }
+);
+
+export const NUMBER_RULES_ASSIGNED_TO_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.all.exceptions.numberRulesAssignedTitle',
+  {
+    defaultMessage: 'Number of rules assigned to',
+  }
+);
+
+export const RULES_ASSIGNED_TO_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.all.exceptions.rulesAssignedTitle',
+  {
+    defaultMessage: 'Rules assigned to',
+  }
+);
+
+export const LIST_DATE_CREATED_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.all.exceptions.dateCreatedTitle',
+  {
+    defaultMessage: 'Date created',
+  }
+);
+
+export const LIST_DATE_UPDATED_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.all.exceptions.dateUPdatedTitle',
```

Comment:
small camelcase issue with `dateUpdatedTitle`

Reply:
Addressed in https://github.com/elastic/kibana/pull/86135